frontend/package.json

```diff
@@ -21,16 +21,19 @@
     "jquery-ui-bundle": "^1.12.1-migrate",
     "jquery.terminal": "^2.1.2",
     "jsonapi-datastore": "^0.4.0-beta",
+    "jszip": "^3.2.2",
     "lint-staged": "^8.2.1",
     "marked": "^0.6.2",
     "material-ui-chip-input": "^1.0.0-beta.16",
     "query-string": "^6.2.0",
     "react": "^16.8.6",
+    "react-circular-progressbar": "^2.0.3",
```

Comment:
Can you update the title / description of this PR to make clear that this PR also adds the progress bar?